### DISCUSSION TITLE
How to use Prepare in React.

### DISCUSSION THREAD

**Discussion by HeisPovedim** [uNmAnNeR/imaskjs#720] [Q&A] [answered]
I can't understand the correct syntax of writing on ReactJS.
the guide says to write like this:
`prepare: function (str) {
    return str.toUpperCase();
  },`
But it doesn't work on recat(

**Comment by karandashovrz** [comment, marked as answer]
@HeisPovedim Check it out https://codesandbox.io/s/react-imask-input-box-masks-forked-tii4v6?file=/src/index.js
Do you have use prepare handler with setted mask property?
